Stitch/Graph/PrototypePreview/Layer/View/CommonModifier/PreviewCommonMisc.swift

```diff
@@ -9,14 +9,34 @@ import Foundation
 import SwiftUI
 import StitchSchemaKit
 
+extension VisibleNodesViewModel {
+    func receivesAPin(_ layerId: NodeId) -> Bool {
+        for layerNode in self.layerNodes {
+            if let layerViewModels = layerNode.value.layerNode?.previewLayerViewModels {
```

Comment:
Slightly more efficient and arguably cleaner:

```suggestion
        for node in self.nodes.values {
            if let layerViewModels = node.layerNode?.previewLayerViewModels {
```

Reply:
I wonder if it might be less efficient?: `self.nodes` is a key-value dict, so `self.nodes.values` probably means Swift first iterated through the dict to grab just the values; then we iterate again through just those values

anyway, this function ended up not being needed after the 'hasPin' perf-fix for using pin-map